src/simplewallet/simplewallet.cpp

```diff
@@ -3456,6 +3456,17 @@ bool simple_wallet::open_wallet(const boost::program_options::variables_map& vm)
     fail_msg_writer() << tr("wallet file path not valid: ") << m_wallet_file;
     return false;
   }
+
+  bool keys_file_exists;
+  bool wallet_file_exists;
+
+  tools::wallet2::wallet_exists(m_wallet_file, keys_file_exists, wallet_file_exists);
+  if(!keys_file_exists)
+  {
+    fail_msg_writer() << boost::format(tr("failed to load wallet: file not found \"%s\"")) % m_wallet_file;
```

Comment:
Still wrong file here.

Reply:
Perhaps instead of showing the path it should just say `Key file not found. Failed to open wallet` to match other error messages previously used?